content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -287,58 +307,62 @@ Invoice creation
    The functional workflow taking place before an invoice validation does **not** alter the main
    changes introduced with the electronic invoice.
 
-Electronic invoices are generated and sent to both the |DIAN| and customer through Carvajal's web
-service integration. These documents can be created from your sales order or manually generated. To
-create a new invoice, go to :menuselection:`Accounting --> Customers --> Invoices`, and select
-:guilabel:`Create`. On the invoice form configure the following fields:
+Electronic invoices are generated and sent to both the |DIAN| and the customer. These documents can
+be created from the sales order or manually generated. To create a new invoice, go to
+:menuselection:`Accounting --> Costumers --> Invoices`, and select :guilabel:`Create`. On the
+invoice form, configure the following fields:
 
-- :guilabel:`Customer`: Customer's information.
-- :guilabel:`Journal`: Journal used for electronic invoices.
+- :guilabel:`Customer`: customer's information.
+- :guilabel:`Journal`: journal used for electronic invoices.
 - :guilabel:`Electronic Invoice Type`: Select the type of document. By default, :guilabel:`Factura
   de Venta` is selected.
 - :guilabel:`Invoice Lines`: Specify the products with the correct taxes.
 
-When done, click :guilabel:`Confirm`.
-
-.. _colombia/invoice-validation:
+.. important::
+   When creating the first invoice related to an electronic invoicing journal, it is required to
+   manually change the *sequence* of the invoice to the |DIAN| format: `Prefix + Sequence`.
 
-Invoice validation
-~~~~~~~~~~~~~~~~~~
+   For example, format the sequence from `SETP1/2024/00001` to `SETP1`.
```

Comment:
@auva-odoo for the example mentioned.
In Odoo, by standard, invoice sequences use the short code from the journal, the year and a chronological sequence (sepereated by "/" symbols). This format is not accepted by the DIAN in Colombia.

This is why it is very important that at the moment of creating the first invoice (where Odoo lets you manually change the sequence) the user changes it to a format that will be accepted by the DIAN which is only the "Prefix + Sequence".

`SETP1/2024/00001` to `SETP1`

To add more information, companies in Colombia need to go through a process where they request their desired "Prefix" to the DIAN, and it needs to be validated: https://www.youtube.com/watch?v=YM_tJgdj4_c

Reply:
```suggestion
   For example, format the sequence from `SETP/2024/00001` to `SETP1`.
```
In fact if we make this change it should be clearer